### ISSUE TITLE
Chamber 18 has issues with aiming and firing Portals

### ISSUE THREAD

**Issue by Gistix** [Gistix/portal2vr#14]
Is it TraceFirePortal again? Sigh...

**Comment by DJanosch** [comment]
I have the problem that I can't set the portal on the ceiling with the bridge in chamber 18.

**Comment by ratcac** [comment]
I have the same problem but cannot create a portal n the ceiling at all I'm assuming this is the big room where the two laser cubes bounce off the faith plates and hit in mid-air and fall to the murky water.

**Comment by jakedowns** [comment]
i ran into this too and eventually got a portal placed by spamming the ceiling plate from the other side of the room until one stuck/spawned

luckily you can shoot through the hard light bridge to get it

at first i thought you couldn't shoot through the bridge and i was making it harder on myself for only aiming for the small corner of white i could see that the bridge was too overlapping

**Comment by ratcac** [comment]
I will try Jakedowns suggestion of firing through the Light bridge and report back.

**Comment by DJanosch** [comment]
yeah, thanks to jakedowns! I was actually able to put a portal on the ceiling through the bridge. To do this, I stood on the platform with the switch all the way to the right and looked straight down. When you can no longer see the platform but only the water below you, you are close enough (at least that's how it was for me). Now just spam the trigger through the bridge until a portal appears (let the cannon circle a bit). and that's it, it worked for me, even 3 times in a row, so it can be reproduced. I don't know what kind of headsets you use, it could be that your fov is different and you can't go as far up the right corner of the platform until you can only see water. approach as far as you can. good luck!

**Comment by jakedowns** [comment]
fwiw, in the main game, with a mouse, you can re-shoot a portal of the same color onto an existing portal to perfect its position. in this mod, it seems like you can no longer do that? (maybe worth a separate ticket)

**Comment by ratcac** [comment]
I finally did it too by following both Jakedowns and DJanosch recommendations - but took about 20 minutes.

**Comment by DanOlner** [comment]
As an alternative, you can skip this chamber onto the next one (rather than going to the start of the next chapter) using the developer console to choose the next chamber.  The code to use and list of chambers is here: https://steamcommunity.com/sharedfiles/filedetails/?id=214866675
